### ISSUE TITLE
The setting only works for single player does not render according to the language file

### ISSUE THREAD

**Issue by erertertet** [maruohon/tweakeroo#497]
As dealing with the translation according to the template in en_US.json in 1.20.x branch, I set the language file of zh_cn.json as:

`
{
    "disableBatSpawning": "测试",
    "config.comment.disablebatspawning": "测试文本",
}`

but in the running demo it have shown something like:

![image](https://github.com/maruohon/tweakeroo/assets/41342153/9e678795-e374-4389-bbd5-a4ecb253ce73)

The same also happened to some other _setting that only apply to single player_ entries

**Comment by sakura-ryoko** [comment]
The single player message is applied a different way, and a bit more hard coded.

Found it:
"tweakeroo.label.config_comment.single_player_only"

**Comment by erertertet** [comment]
I meant the "disablebatSpawning" string, that is not replaced by the string specified in language file

**Comment by maruohon** [comment]
Looks like you may actually want to use this format instead, or rather additionally, for the config names:
![image](https://github.com/maruohon/tweakeroo/assets/2331050/93494fc3-2a55-4b6a-8934-40663b17085b)

This stuff is somewhat of a mess in the old code, as some stuff uses `getPrettyName()` and some stuff uses `getConfigGuiDisplayName()`.
![image](https://github.com/maruohon/tweakeroo/assets/2331050/13543d22-97af-4c11-9f25-27e14796ddd3)
![image](https://github.com/maruohon/tweakeroo/assets/2331050/3e4d6b61-4d4d-4c3d-ba35-102045d9c17b)

So basically I think you want to provide the translation both for `disableBatSpawning` and also `config.name.disablebatspawning`.
And the latter will be made as that gold/orange text automatically for those "single player only" configs in Tweakeroo.
![image](https://github.com/maruohon/tweakeroo/assets/2331050/a3970415-7f12-4ac4-9c56-1ffc14e17660)

**Comment by erertertet** [comment]
It kind of worked out if I just use all the config name translation by just do

`{"config.name.aaabbbccc": "<translation>"}`

instead of 

`{"aaaBbbbCcc": "<translation>"}`

![image](https://github.com/maruohon/tweakeroo/assets/41342153/158564aa-b86c-459b-b1f2-97df1040904e)

should I use both of them or just use one of them as the translation.json?

**Comment by maruohon** [comment]
I think you need the short form for any of the toggle messages to be translated at least.
